pkg/controllers/user/cis/clusterScanHandler.go

```diff
@@ -116,43 +143,44 @@ func (csh *cisScanHandler) Updated(cs *v3.ClusterScan) (runtime.Object, error) {
 	return cs, nil
 }
 
-func (csh *cisScanHandler) deployApp(clusterName, appName string) error {
+func (csh *cisScanHandler) deployApp(appInfo *appInfo) error {
 	appCatalogID := settings.SystemCISBenchmarkCatalogID.Get()
 	err := utils.DetectAppCatalogExistence(appCatalogID, csh.mgmtCtxTemplateVersionLister)
 	if err != nil {
 		return errors.Wrapf(err, "cisScanHandler: deployApp: failed to find cis system catalog %q", appCatalogID)
 	}
-	appDeployProjectID, err := utils.GetSystemProjectID(clusterName, csh.projectLister)
+	appDeployProjectID, err := utils.GetSystemProjectID(appInfo.clusterName, csh.projectLister)
 	if err != nil {
 		return err
 	}
 
-	appProjectName, err := utils.EnsureAppProjectName(csh.userCtxNSClient, appDeployProjectID, clusterName, DefaultNamespaceForCis)
+	appProjectName, err := utils.EnsureAppProjectName(csh.userCtxNSClient, appDeployProjectID, appInfo.clusterName, v3.DefaultNamespaceForCis)
 	if err != nil {
 		return err
 	}
 
-	creator, err := csh.systemAccountManager.GetSystemUser(clusterName)
+	creator, err := csh.systemAccountManager.GetSystemUser(appInfo.clusterName)
 	if err != nil {
 		return err
 	}
 
 	appAnswers := map[string]string{
-		"owner": appName,
+		"owner": appInfo.appName,
+		"skip":  appInfo.skip,
 	}
 
 	app := &projv3.App{
 		ObjectMeta: metav1.ObjectMeta{
 			Annotations: map[string]string{creatorIDAnno: creator.Name},
-			Name:        appName,
+			Name:        appInfo.appName,
 			Namespace:   appDeployProjectID,
 		},
 		Spec: projv3.AppSpec{
 			Answers:         appAnswers,
```

Comment:
as long as passing "skip" will do what you want later then the rest of this is good. merge types and update go mod for new hashes then it's lgtm